From a8c8b39ce42b9360ee5419b1346e9d8899693bfc Mon Sep 17 00:00:00 2001
From: AppaRao Puli <apparao.puli@linux.intel.com>
Date: Thu, 24 Sep 2020 01:12:11 +0530
Subject: [PATCH] Add missing Context property to MetricReport

MetricReport schema version 1.3.1 added Context
property which is missing in implementation.

Tested:
 - Recieved MetricReport data on Listner has
   context in it.

Change-Id: Iafd37902b2295c9ed91e7b68e60303b37d32a89f
Signed-off-by: AppaRao Puli <apparao.puli@linux.intel.com>
---
 redfish-core/include/event_service_manager.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/redfish-core/include/event_service_manager.hpp b/redfish-core/include/event_service_manager.hpp
index 22d1f10..fd42985 100644
--- a/redfish-core/include/event_service_manager.hpp
+++ b/redfish-core/include/event_service_manager.hpp
@@ -548,6 +548,7 @@ class Subscription
             {"@odata.type", "#MetricReport.v1_3_0.MetricReport"},
             {"Id", id},
             {"Name", id},
+            {"Context", this->customText},
             {"Timestamp", readingsTs},
             {"MetricReportDefinition", {{"@odata.id", metricReportDef}}},
             {"MetricValues", metricValuesArray}};
-- 
2.7.4

